jdk8/plan.sh

```diff
@@ -72,7 +72,7 @@ do_install() {
     -exec sh -c 'file -i "$1" | grep -q "x-executable; charset=binary"' _ {} \; \
     -exec patchelf --interpreter "$(pkg_path_for glibc)/lib/ld-linux-x86-64.so.2" --set-rpath "$LD_RUN_PATH" {} \;
 
-  find $pkg_prefix/lib/amd64/*.so $pkg_prefix/jre/lib/amd64 -type f \
+  find "$pkg_prefix"/lib/amd64/*.so "$pkg_prefix"/jre/lib/amd64 -type f \
```

Comment:
This line stood out to me too when I did my own version of this PR, I think it should be: 

```
find "$pkg_prefix/lib/amd64" "$pkg_prefix/jre/lib/amd64" -type f -name "*.so" \
```